File: LSH0707/BOJ/boj28702.py
a = input()
b = input()
c = input()
if a.isdigit() == True: # 입력값 3개중 최소 1개는 숫자이므로 다음에올 숫자 찾기
    d = int(a) + 3
elif a.isdigit() == False and b.isdigit() == True:
    d = int(b) + 2
elif a.isdigit() == False and b.isdigit() == False and c.isdigit() == True:
    d = int(c) + 1

if d % 15 == 0:
    print('FizzBuzz')
elif d % 5 == 0:
    print('Buzz')
elif d % 3 == 0:
    print('Fizz')
else:
    print(d)

File: SngJuni/BOJ/boj28702.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    int pos = 0, num = 0;

    string s;
    for (int i = 0; i < 3; i++) {
        cin >> s;  // 문자열로 입력받음

        if (s == "Fizz" || s == "Buzz" || s == "FizzBuzz") continue;
        else {     // 3개의 입력 중 하나는 반드시 숫자가 들어오기 때문에 이를 문자열에서 int 형으로 변환
            pos = i;
            num = stoi(s);
        }
    }

    num += (3 - pos);  // 다음 올 문자열에 해당하는 숫자
    if (!(num % 3) && !(num % 5)) cout << "FizzBuzz\n";
    else if (!(num % 3)) cout << "Fizz\n";
    else if (!(num % 5)) cout << "Buzz\n";
    else cout << num << '\n';

    return 0;
}